libs/remix-ui/editor/src/lib/providers/codeActionProvider.ts

```diff
@@ -0,0 +1,48 @@
+import { Monaco } from "@monaco-editor/react"
+import monaco from "../../types/monaco"
+import { EditorUIProps } from "../remix-ui-editor"
+import { default as fixes } from "./quickfixes"
+
+export class RemixCodeActionProvider implements monaco.languages.CodeActionProvider {
+    props: EditorUIProps
```

Comment:
Please use one space as a tab

Reply:
the default for TS is 4 spaces

Reply:
https://github.com/ethereum/remix-project/pull/3939 sets all to 2 spaces. 1 is not best practice.

Reply:
As linting is being fixed for different PR, can you review again? @LianaHus